package/doc/sphinx/source/documentation_pages/analysis/parallelization.rst

```diff
@@ -41,7 +41,8 @@ impossible with any but the ``serial`` backend.
 
    Parallelization is getting added to existing analysis classes.  Initially,
    only :class:`MDAnalysis.analysis.rms.RMSD` supports parallel analysis, but
-   we aim to increase support in future releases.
+   we aim to increase support in future releases. Please check issues labeled
+   `parallelization` on the MDAnalysis issues [page](https://github.com/MDAnalysis/mdanalysis/issues).
```

Comment:
You can make the link https://github.com/MDAnalysis/mdanalysis/issues?q=is%3Aopen+is%3Aissue+label%3Aparallelization